conda_recipe/meta.yaml

```diff
@@ -57,6 +58,7 @@ requirements:
     - libxml2
     - libcypher-parser
     - nlohmann_json
+    - openblas
```

Comment:
If we're not changing public repo code I don't think the conda dependencies should change, unless I'm missing something.

Reply:
Correct. If we want to make a conda `katana-enterprise` package we could (and I suspect we may want to eventually). But for now the conda is open only. 

The documentation is currently wrong about this. And there are actually several things in the conda docs that are only used in enterprise. I need to do a pass and fix those issues.